storage/innobase/xtrabackup/src/ds_encrypt.c

```diff
@@ -552,6 +555,12 @@ encrypt_worker_thread_func(void *arg)
 		if (thd->cancelled)
 			break;
 
+		assert(gcry_md_get_algo_dlen(XB_CRYPT_HASH) ==
```

Comment:
Yes I understand that, I just meant that a source code comment would have been appreciated here e.g.

```c
#Ensure the returned value of gcry_md_get_algo_dlen(XB_CRYPT_HASH) is equal to XB_CRYPT_HASH_LEN or throw an assertion.
assert(gcry_md_get_algo_dlen(XB_CRYPT_HASH) ==
		       XB_CRYPT_HASH_LEN);
```

Entirely depends on your coding standard though, so feel free to not include something like this.